### ISSUE TITLE
bgp.test_bgp_gr_helper.test_bgp_gr_helper_routes_perserved failed

### ISSUE THREAD

**Issue by lguohan** [Azure/sonic-mgmt#1906]
**Description**

**Steps to reproduce the issue:**
1. kvm test https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/sonic-mgmt-testing/job/sonic-mgmt-pr/438/testReport/junit/bgp/test_bgp_gr_helper/test_bgp_gr_helper_routes_perserved/
2.
3.

**Describe the results you received:**
```
            # parse set_src
>           m = re.match(r"^default proto (bgp|186) src (\S+)", rt[0])
E           IndexError: list index out of range
```

**Describe the results you expected:**
test pass.

**Additional information you deem important:**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by lguohan** [comment]
looking at the log. after setup, the default is empty.

```
DEBUG    root:devices.py:65 /data/sonic-mgmt/tests/common/devices.py::get_ip_route_info#572: [vlab-01] AnsibleModule::command, args=["ip route list match 0.0.0.0"], kwargs={}
DEBUG    root:devices.py:79 /data/sonic-mgmt/tests/common/devices.py::get_ip_route_info#572: [vlab-01] AnsibleModule::command Result => {"stderr_lines": [], "cmd": ["ip", "route", "list", "match", "0.0.0.0"], "end": "2020-07-15 07:23:27.561717", "_ansible_no_log": false, "stdout": "", "changed": true, "rc": 0, "start": "2020-07-15 07:23:27.554122", "stderr": "", "delta": "0:00:00.007595", "invocation": {"module_args": {"warn": true, "executable": null, "_uses_shell": false, "strip_empty_ends": true, "_raw_params": "ip route list match 0.0.0.0", "removes": null, "argv": null, "creates": null, "chdir": null, "stdin_add_newline": true, "stdin": null}}, "stdout_lines": []}
INFO     root:devices.py:576 route raw info for 0.0.0.0: []
```

**Comment by wangxin** [comment]
According to the exabgp err log, the exabgp process in PTF was restarted somehow during testing and caused this issue.
Checked the Jenkins workers, there were some old `scripts/vs/buildimage-vs-image/test.sh` still running. Those scripts were started on Jun 23. Not sure if the exabgp restart was related with those long running scripts. After I killed them, then this issue was not observed since build 440 so far.
